### PR TITLE
feat: customize sort function

### PR DESCRIPTION
**Pull request by reclay**
customize sort function

### CODE REVIEW

**Comment by njleonzhang** [comment]
Tnx for PR. great performance improvement and beautiful use of ternary expression. I will add unit test for this feature later. Thank you again.

File: src/components/DataTables.vue [hunk 1]
```diff
@@ -63,20 +63,8 @@
...
-              return -1
-            } else {
-              return 0
-            }
-          })
-          if (isDescending) {
-            sortedData.reverse()
-          }
+          let _sortMethod = (a, b) => isDescending ? -this.sortMethod(a[prop], b[prop]) : this.sortMethod(a[prop], b[prop])
+          sortedData.sort(_sortMethod)
```
Comment (njleonzhang) [inline src/components/DataTables.vue]: beatiful!

File: src/mixins/ShareMixin.js [hunk 2]
```diff
@@ -62,6 +62,10 @@ export default {
       default() {
         return {}
       }
+    },
+    sortMethod: {
```
Comment (njleonzhang) [inline src/mixins/ShareMixin.js]: sortMethod is not  related to vue-data-tables-server. so this should be added in `DataTables.vue`

Comment (reclay) [reply src/mixins/ShareMixin.js]: Thanks.

**Pull request merged**